docs/en/start/rpc/dubbo_json_rpc.md

```diff
@@ -20,17 +20,12 @@ git clone git@github.com:apache/dubbo-go-samples.git
 cd dubbo-go-samples
 
 # start zookeeper
-cd attachment/go-server
-make -f ../../build/Makefile docker-up 
-cd -
-
-# build dubbo server
-cd general/jsonrpc/go-server
-sh assembly/mac/dev.sh
+cd rpc/jsonrpc/go-server
+docker-compose -f docker/docker-compose.yml up -d
 
-# start dubbo server
-cd target/darwin/{generate_folder}/
-sh ./bin/load.sh start
+# build && start dubbo server
+cd cmd
+go run .
```

Comment:
I met another error when running this line:
```shell 
admindeMacBook-Pro-2:cmd qunli$ go run .
panic: open /Users/qunli/projects/layotto/dubbo-go-samples/rpc/jsonrpc/go-server/cmd/conf/dubbogo.yaml: no such file or directory

goroutine 1 [running]:
dubbo.apache.org/dubbo-go/v3/config.NewLoaderConf({0x0, 0x0, 0xc000068738})
        /Users/qunli/go/pkg/mod/dubbo.apache.org/dubbo-go/v3@v3.0.2-0.20220316080055-f481b16d363c/config/config_loader_options.go:80 +0x1bc
dubbo.apache.org/dubbo-go/v3/config.Load({0x0, 0x4e9a0c0, 0xc0000001a0})
        /Users/qunli/go/pkg/mod/dubbo.apache.org/dubbo-go/v3@v3.0.2-0.20220316080055-f481b16d363c/config/config_loader.go:45 +0x25
main.main()
        /Users/qunli/projects/layotto/dubbo-go-samples/rpc/jsonrpc/go-server/cmd/server.go:47 +0x25
exit status 2
```

It seems that we should set some environment variable before running it, see https://github.com/apache/dubbo-go-samples/blob/f0d1e1076397a4736de080ffb16cd0963c8c2f9d/rpc/jsonrpc/go-server/cmd/server.go#L44

Could u please add a command above to set the env config?

For example:
```
cd cmd
export DUBBO_GO_CONFIG_PATH=../conf/dubbogo.yml
go run .
```

Reply:
By the way, after setting the env config, starting the server ,and sending the request,  I got no response ...
![image](https://user-images.githubusercontent.com/26001097/162767814-1df720e3-39a2-405d-8be8-043515d43811.png)

It needs some debug here

Reply:
Oh, I can see the result now:
![image](https://user-images.githubusercontent.com/26001097/162915569-486c0821-646d-4031-9937-081d10431d23.png)